server/models/test/data.sql

```diff
@@ -13,7 +13,8 @@ INSERT INTO unit VALUES
 (42,  'Project','TREE.PROJECT','',1,'/partials/projects/','/projects'),
 (48,  'Service Management','TREE.SERVICE','',1,'partials/services/','/services'),
 (80,  'Patient Update','TREE.EDIT_PATIENT','',12,'/partials/patient_edit/','/patients/edit/'),
-(105, 'Cashbox Management','TREE.CASHBOX_MANAGEMENT','',1,'/partials/cash/cashbox/','/cashboxes');
+(105, 'Cashbox Management','TREE.CASHBOX_MANAGEMENT','',1,'/partials/cash/cashbox/','/cashboxes'),
+(106, 'Depot Management', 'TREE.DEPOTS_MANAGEMENT', 'Depot Management module', 1, '/partials/depots_management/', '/depots_management');
```

Comment:
Can you add a permission record for this, so that the super user has access to this module?